manage

```diff
@@ -499,12 +499,14 @@ if ! isInstalled ${SED_EXE}; then
 
   cat <<-EOF
   The recommended approach to installing the required package(s) is to use either [Homebrew](https://brew.sh/) (MAC)
-  or [Chocolatey](https://chocolatey.org/) (Windows).  For more information visit https://mikefarah.github.io/yq/
+  or [Chocolatey](https://chocolatey.org/) (Windows) or [GitHub](https://github.com/mirror/sed) (Linux).  For more information visit https://mikefarah.github.io/yq/
 
   Windows:
     - chocolatey install sed
   MAC:
     - brew install gnu-sed
+  Linux (Ubuntu):
+    - apt install sed
```

Comment:
```suggestion
  Debian/Ubuntu:
    - sudo apt-get install sed
```

Reply:
I'm making this recommendation to keep things consistent with the changes you made in `fetch-validator-status/run.sh`